index.html

```diff
@@ -123,7 +129,7 @@ <h3 class="work-name">Multi-Post Stories</h3>
                         <li class="lang">ruby</li>
                     </ul>
                 </div>
-                <button class="see-project">See Project</button>
+                <button id="see-project" class="see-project">See Project</button>
```

Comment:
The pop-up window should cover almost the whole screen width and height on the mobile viewport, but it's aligned to the right side of the project. Moreover, buttons at the bottom also can not be seen. Kindly see the screenshot below and fix the issues accordingly.
![mobile](https://user-images.githubusercontent.com/35504149/132556394-5bb00a1a-0650-4d04-982f-308b23f3a7dc.png)